# Predicting Protein Secondary Structure

![Image](https://www.ebi.ac.uk/training/online/courses/biomacromolecular-structures/wp-content/uploads/sites/110/2020/09/Figure13-1024x973.png)


### Introduction
"Proteins are linear polymeric chains made of protein-specific sequences of 20 amino acid residue types. Proteins can perform a wide variety of molecular functions ranging from molecular recognition, catalysis, molecular motors, to structural support, in part because they can fold into many different three-dimensional structural shapes. Thus, understanding how proteins function requires knowledge of their structures"

"predicted probabilities of secondary structures in those intrinsically unstructured or disordered proteins (i.e. absence of unique tertiary structures) can provide clues for functional sites in their unstructured regions by binding or induced folding. 
In addition, because structures are more conserved than sequences and structures determine functions, predicted secondary structures are proven useful in protein sequence alignment [and protein function prediction. 
Owing to the importance of secondary structure in protein structure stability and function, disease-causing mutations are often located in regions with secondary structures. 
As a result, predicted secondary structures are an important feature in the methods for discriminating disease-causing from neutral genetic variations"
From: [Article](https://academic.oup.com/bib/article/19/3/482/2769436?login=false)

### Goal
Create model to predict secundary structure from sequence of Aminoacids.

### Sources

[Dataset Source](https://webs.iiitd.edu.in/raghava/ccpdb/compile.php)
[Kaggle Source](https://www.kaggle.com/datasets/tamzidhasan/protein-secondary-sequence)